silkrpc/commands/web3_api.cpp

```diff
@@ -29,8 +29,16 @@ namespace silkrpc::commands {
 
 // https://eth.wiki/json-rpc/API#web3_clientversion
 asio::awaitable<void> Web3RpcApi::handle_web3_client_version(const nlohmann::json& request, nlohmann::json& reply) {
-    reply = make_json_content(request["id"], common::kEthereumNodeName);
```

Comment:
Now you can delete the following lines in file `constants.hpp`:
```
constexpr const char* kNodeName{"TurboGeth"};
constexpr const char* kVersion{"2021.05.3-alpha"};
const std::string kEthereumNodeName = boost::str(boost::format("%1%/v%2%/%3%") % kNodeName % kVersion % kOsName); // NOLINT

constexpr uint64_t kETH66{66};
```

Reply:
ok